src/swap/components/SwapSettingsSlippageInput.test.tsx

```diff
@@ -74,26 +68,6 @@ describe('SwapSettingsSlippageInput', () => {
     expect(screen.getByRole('textbox')).toBeDisabled();
   });
 
-  it('switches between Auto and Custom modes', () => {
```

Comment:
do we have replacement tests for using the values from the config? and why did we remove this test?

Reply:
These tests were using the old defaultSlippage input param. I was going to remove and write new ones but we are still getting 100% test coverage 👀